release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -72,6 +72,32 @@ Previously, secret information was used as the OAuth access token and OAuth auth
 If you are upgrading your cluster to {product-title} 4.6, old tokens from {product-title} 4.5 will still have the secret information exposed in the object name. By default, the expiration for tokens is 24 hours, but this setting can be changed by administrators. Sensitive data can still be exposed until all old tokens have either expired or have been deleted by an administrator.
 ====
 
+[id="ocp-4-6-networking"]
+=== Networking
+
+[id="ocp-4-6-pod-network-connectivity-checks"]
+==== Pod network connectivity checks
+
+Operators can now configure `PodNetworkConnectivityCheck` resources to check each network connection from the Pods that are managed by the Operator. This allows you to more easily identify and troubleshoot issues with important network connections in your cluster.
```

Comment:
@bergerhoffer, this is implemented by a CRD:

From `oc api-resources`:

```
podnetworkconnectivitychecks                           controlplane.operator.openshift.io    true         PodNetworkConnectivityCheck
```

So as of today at least, wouldn't this not use back ticks and s/resources/custom resources/?

Reply:
@jboxman technically yes, but I tend to just use "resource" because I don't think the distinction is important.